v2/cmd/generate.go

```diff
@@ -198,7 +201,8 @@ func init() {
 	generateCmd.Flags().BoolVar(&generateCmdOpts.FromDDL, "from-ddl", false, "toggle using ddl file")
 	generateCmd.Flags().StringVarP(&generateCmdOpts.Out, "out", "o", "", "output path or file name")
 	generateCmd.Flags().StringVar(&generateCmdOpts.Suffix, "suffix", defaultSuffix, "output file suffix")
-	generateCmd.Flags().StringVarP(&generateCmdOpts.Package, "package", "p", "", "package name used in generated Go code")
+	generateCmd.Flags().StringVarP(&generateCmdOpts.PackagePath, "package-path", "p", "", "Go package import path for the generated Go code")
+	generateCmd.Flags().StringVar(&generateCmdOpts.PackageName, "package-name", "", "Go package name for the generated Go code")
```

Comment:
The `package-name` is the same as the old `package` flag. If users omit it, we assume the last element of the package path corresponds to its package name.